sdk-lib/src/main/java/cash/z/ecc/android/sdk/Synchronizer.kt

```diff
@@ -110,6 +112,11 @@ interface Synchronizer {
     // Operations
     //
 
+    suspend fun getFastestServers(
```

Comment:
Please provide at least basic Javadoc documentation here, as it's a new member of the SDK's public API.